packages/salesforcedx-vscode-soql/package.json

```diff
@@ -35,20 +35,16 @@
   "dependencies": {
     "@salesforce/apex-tmlanguage": "1.6.0",
     "@salesforce/core": "^2.15.2",
-    "@salesforce/salesforcedx-utils-vscode": "51.4.0",
     "@salesforce/soql-builder-ui": "0.1.0",
-    "@salesforce/soql-common": "0.2.1",
     "@salesforce/soql-data-view": "0.1.0",
     "@salesforce/soql-language-server": "0.6.2",
-    "@salesforce/soql-model": "0.2.1",
-    "applicationinsights": "1.0.7",
-    "debounce": "^1.2.0",
-    "jsforce": "^1.10.0",
-    "papaparse": "^5.3.0",
-    "vscode-languageclient": "6.1.3"
+    "jsforce": "^1.10.0"
   },
```

Comment:
Dependencies that are webpacked with the SOQL extension module are moved to `devDependencies`. 

Dependencies that are not webpacked: 
- `@salesforce/apex-tmlanguage`: a path to a file in this package is referenced from the SOQL extension which is not resolved when webpacked
- `@salesforce/soql-builder-ui`: a path to a file in this package is referenced from the SOQL extension which is not resolved when webpacked
- `@salesforce/soql-data-view`: a path to a file in this package is referenced from the SOQL extension which is not resolved when webpacked
- `@salesforce/soql-language-server`: a path to a module in this package is calculated and executed from the SOQL extension which is not resolved when webpacked
- `@salesforce/core`: a path to an internal file is `require`'d in this package which is not resolved when webpacked
- `jsforce`: a common dependency of the SOQL extension and `@salesforce/core`; webpacking would include this package twice in the resulting VSIX; once in the webpack and once externally as a dependency of `@salesforce/core`